portal/static/js/main.js

```diff
@@ -4222,6 +4222,42 @@ var tnthTables = {
         if (isNaN(b_d)) b_d = 0;
 
         return  b_d - a_d;
+    },
+    /***
+     * sorting alpha numeric string
+     */
+     "alphanumericSorter": function (a, b) {
+        /*
+         * ideas came from
+         * http://www.davekoelle.com/files/alphanum.js
```

Comment:
Would it make sense to keep this as a separate file? I found it copied as [a gist here](https://gist.github.com/myadzel/6405e60256df579eda8c). looks like those work with rawgit too :)

https://cdn.rawgit.com/myadzel/6405e60256df579eda8c/raw/e24a756e168cb82d0798685fd3069a75f191783f/alphanum.js